docs/source/guide/upgrading.rst

```diff
@@ -4,7 +4,7 @@ Upgrading Notes
 
 Notes to refer to when upgrading ``boto3`` versions.
 
-1.4.0
+1.4.x
```

Comment:
That works to. I will update along with making the floor dependency of s3transfer to 0.1.10